eventmesh-connector-plugin/build.gradle

```diff
@@ -13,25 +13,4 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
-
-task copyConnectorPlugin(dependsOn: ['jar']) {
-    doFirst {
-        new File(projectDir, '../eventmesh-connector-plugin/dist/apps').mkdir()
-        new File(projectDir, '../dist/plugin/connector').mkdirs()
```

Comment:
may not need dist directory, just $HOME/plugin/connector

Reply:
Now this task has been removed, we can use the `dist` task to insatll all plugin, the plugin directory will look like
![image](https://user-images.githubusercontent.com/22415594/135755983-6b6c52a6-12f8-4843-8062-d939accfb066.png)